Resources/Prototypes/DeltaV/Entities/Structures/Piping/Disposal/space_disposal.yml

```
@@ -0,0 +1,25 @@
+- type: entity
+  id: SpaceDisposalUnit
+  parent: DisposalUnit
+  name: space disposal unit
+  description: A pneumatic waste disposal unit with warning signs telling you that it leads to space and that NT is not responsible for any injury.
+  components:
+  - type: Sprite
+    sprite: DeltaV/Structures/Piping/disposal.rsi
+    snapCardinals: true
+    layers:
+    - state: condisposal
+      map: [ "enum.DisposalUnitVisualLayers.Unanchored" ]
+    - state: disposal
+      map: [ "enum.DisposalUnitVisualLayers.Base" ]
+    - state: disposal-flush
+      map: [ "enum.DisposalUnitVisualLayers.BaseFlush" ]
```

```suggestion
      map: [ "enum.DisposalUnitVisualLayers.OverlayFlush" ]
```
this should fix test errors